components/TransactionItem/TransactionItem.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from '../TransactionItem/TransactionItem.module.css';
+import { ListItem } from "@mui/material";
+
+type TransactionItemProps = {
+    date: Date,
+    username: string,
+    fid: string,
+    admin: string,
+    message: string,
+    change: number
+};
+
+function twoDecimal(yourNumber){
+    return Math.round(yourNumber * 100) / 100;
+}
+
+export const TransactionItem: React.FunctionComponent<TransactionItemProps> = ({
+    date,
+    username,
+    fid,
+    admin,
+    message,
+    change,
+}) => {
+    return (
+        <ListItem className={styles['list-item']}>
+            <div className={styles['date']}>{date.toLocaleDateString("en-US")}</div>
+            <div className={styles['username']}>{username}</div>
+            <div className={styles['fid']}>{fid}</div>
+            <div className={styles['admin']}>{admin}</div>
+            <div className={styles['action']}>
+                {
+                    change>0?
+                    <div className={styles['earn-action']}>Earn</div>:
+                    <div className={styles['redeem-action']}>Redeem</div>
+                }
+            </div>
+            <div className={styles['message']}>{message}</div>
```

Comment:
I tried playing around with the message displayed in this component, and saw this:
<img width="1394" alt="Screen Shot 2021-10-03 at 10 36 55 PM" src="https://user-images.githubusercontent.com/28970991/135799490-8576b677-758d-4e86-ad47-83c8b5c69675.png">

Please fix the styling on this div so the text wraps around! It's very possible that admins will input long messages/descriptions, and we wanna make sure that it's displayed nicely!